src/main/java/menion/android/whereyougo/network/activity/DownloadCartridgeActivity.java

```diff
@@ -50,9 +50,11 @@
 
 public class DownloadCartridgeActivity extends CustomActivity
         implements NoPasswordDialogFragment.NoPasswordDialogListener {
-    private static final String TAG = "DownloadCartridgeActivity";
+    private static final String TAG = "DownloadCartridgeAct";
     private DownloadCartridgeTask downloadTask;
     private String cguid;
+    private String Username;
+    private String Password;
```

Comment:
The variable names should be in lower case.

Reply:
Next commit will fix that. I will push it together with the other changes from your review.

Reply:
Is not yet changed.